### PR TITLE
feat(Notifications Modal): Added notifications modal

### PR DESCRIPTION
**Pull request by wski**

### CODE REVIEW

**Comment by wski** [comment]
@clarkjefcoat @akonsowski @SOAP1914

**Comment by akonsowski** [comment]
@bhalt the syles look a little funky any idea whats causing this?
image

other than the look :+1:

File: app/js/components/header/notificationsmodal.jsx [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+        var date = new Date(n.createdDate);
+        var months = ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'June', 'July', 'Aug', 'Sept', 'Oct', 'Nov', 'Dec'];
+        var formattedDate = (months[date.getMonth() + 1]) + ' ' + date.getDate() + ', ' +  date.getFullYear();
+        return (
+          <li role="presentation" alt={`Notification ${i + 1} from ${n.author.user.username}`} tabIndex={i} onClick={() => {
+              this.setState({
+                activeNotification: i
+              });
+            }}>
+            <a href="#" onClick={(e) => {e.preventDefault()}}>
```
Comment (clarkjefcoat) [inline app/js/components/header/notificationsmodal.jsx]: jslint complains about a missing semicolon

**Comment by clarkjefcoat** [comment]
Please fix the jslint error.  Ditto on akonsowski's comment.  :+1:

**Comment by bhalt** [comment]
Can we change the `right` class on the Remove Notification button to `pull-right` so it fits into the global bootstrap and works on HUD? @wski

**Comment by bhalt** [comment]
Also, I'm updating the global ozp-bootstrap-sass to accommodate nav-tabs/pills in the dark theme :+1: good catch

**Pull request merged**